.github/workflows/ci.yml

```diff
@@ -91,10 +87,11 @@ jobs:
       - name: Error Prone Checks
         run: |
           export MAVEN_OPTS="${MAVEN_INSTALL_OPTS}"
+          # gib is explicitly disabled since we want to ensure errorprone passes on all modules
           # Run Error Prone on one module with a retry to ensure all runtime dependencies are fetched
-          $RETRY $MAVEN ${MAVEN_TEST} -T C1 clean test-compile -P gib,errorprone-compiler -pl ':trino-spi'
+          $RETRY $MAVEN ${MAVEN_TEST} -T C1 clean test-compile -Dgib.disable -P errorprone-compiler -pl ':trino-spi'
```

Comment:
Can you also remove this from `MAVEN_TEST` and only activate the `gib` profile in the `test` job? This will also require removing `-Dgib.disable` from product test jobs.

Reply:
Why? To avoid unknowingly introducing gib in a step?

Reply:
Pushed a fixup. PTAL.